docs/ota-client-guide/modules/ROOT/pages/garage-sign-reference.adoc

```diff
@@ -0,0 +1,357 @@
+= Garage-sign commands and options
+
+You can use the `garage-sign` tool if you need to sign metadata about your software, as well as manage your software signing keys and root of trust. When you start using OTA Connect, we generate a root of trust and a signing key for you, and automatically sign software you upload. Later on, you can use `garage-sign` to rotate those keys, taking them completely offline to increase security.
+
+In the reference below, find a list of commands and options that you can use.
+
+.Garage-sign reference
++++<details><summary>+++
+`user-keys [gen|id]`—Manages keys that are not associated with a specific repository.
++++ </summary><div> +++
+
++++<details><summary>+++
+`user-keys gen`—Creates key pairs that are not associated with a specific repository.
++++</summary><div>+++
+
+[.release_notes]
+[cols="15m,75a"]
+|====================
+| -t, --type | The type of key that you want to create: `ed25519` or `rsa`.
+| --keysize | The length of the key that you want to create, in bits. RSA 2048/4096 an Ed25519 are supported.
+| -k, --key-name | The name of the file with your private and public keys.
+|====================
+
++++</div></details>+++
+
++++<details><summary>+++
+`user-keys id`—Calculates your public key ID.
++++</summary><div>+++
+
+[.release_notes]
+[cols="15m,75a"]
+|====================
+| -i, --input | The path to the file with your public keys.
+|====================
+
++++</div></details>+++
+
+[.release_notes]
+[cols="15m,75a"]
+|====================
+| -p, --keys-path | The path where this executable will look for keys. By default, it is the `user-keys` directory in `home-dir`.
+|====================
+
++++</div></details>+++
+
++++<details><summary>+++
```

Comment:
I am not 100% sure about the fully functional part of the delegation part of garage-sign. I have been trough this epic (OTA-677) on which we have been working on last year: I don't recall whether it was end-2-end tested and whether all the changes were deployed to production.
I would suggest to flag this part as 'Beta'. @patrickvacek , @simao  what do you think ?

Reply:
It works last time I tried it. We still maintain unit tests for all of the delegation support in aktualizr. There are some limitations but it works as far as I know.

Reply:
To be on the safe side, should we add the Beta tag? Or is it okay to keep it as is?

Reply:
@xaviergully  @patrickvacek 
I added the **Beta** tag to delegations just to be on the safe side :)